viaq-jenkins/fields.yml

```diff
@@ -291,17 +298,20 @@ _default_:
       type: string
       index: not_analyzed
       description: >
-        name of the log file. 
+        Optional field present when the data is collected from a file from a known file location.
+        The field may contain the full local path to the file.
 
     - name: offset
       type: long
       description: >
-        offset of the message in the logs file. 
+        Offset of the message in the logs file(see the field 'file'). 
 
 jenkins_data:
   type: group
   description: >
     RHCI-specific mapping. It inherits all the fields from _default_ mapping
+    This is the name you put in _type when ingesting the documents in ES
```

Comment:
this is the name of the mapping.
this will be the value used as the ElasticSearch Document Type of a document being indexed.